x-pack/test/apm_api_integration/tests/alerts/transaction_duration.spec.ts

```diff
@@ -110,49 +92,43 @@ export default function ApiTest({ getService }: FtrProviderContext) {
               group: 'threshold_met',
               id: actionId,
               params: {
-                documents: [{ message: 'Transaction Name: {{context.transactionName}}' }],
+                documents: [
+                  { message: 'Transaction Name: {{context.transactionName}}', id: actionId },
+                ],
               },
               frequency: {
                 notify_when: 'onActionGroupChange',
-                throttle: null,
                 summary: false,
               },
             },
           ],
         });
+
         expect(createdRule.id).to.not.eql(undefined);
+
         ruleId = createdRule.id;
-      });
 
-      it('checks if rule is active', async () => {
-        const executionStatus = await waitForRuleStatus({
+        const executionStatus = await alertTestHelper.waitForRuleStatus({
           id: ruleId,
           expectedStatus: 'active',
```

Comment:
Do we ever want to wait for anything but an active rule?

```suggestion
        const executionStatus = await alertTestHelper.waitForActiveRule({
          id: ruleId
```